README.md

```diff
@@ -140,7 +140,8 @@ The following settings are stored in `template_config.yml`.
   pydocstyle            Boolean, whether to have flake8 use pydocstyle to check for compliance with
                         Python docstring conventions.
 
-  python_version        Python version to use in the CI. Currently only 3.6 and 3.8 are supported.
+  python_version        Python version to use in the CI. This does not affect the python version
+                        inside the CI container.
```

Comment:
We can drop this line now too. Thank you for removing one more question for a useless user decision.